### DISCUSSION TITLE
Install Directly to External HDD

### DISCUSSION THREAD

**Discussion by derspion318** [iiab/iiab#3350] [Q&A] [answered]
Howdy, all!
     I am having an issue. I followed the instructions in FAQ for "Can I permanently attach an external USB drive, to add more content?", but this still will not allow me to install anything larger than the size of my SD card. I am using a smaller SD card because I have a multi-TB external drive attached to the raspberry pi. How does one make it where the default install directory is the external drive?

**Comment by holta** [comment, marked as answer]
Consider mounting your external disk prior to installing IIAB?

1) One common way to do this is to
configure [/etc/fstab](https://en.m.wikipedia.org/wiki/Fstab) to mount your external drive as `/library`

   (In other words, put IIAB software on the "internal" microSD card — and then add your content in `/library` as usual — it's just that `/library` happens to be on your external drive.)

2) Another way to arrange everything is to avoid using a microSD card entirely — booting directly from the external drive (e.g. Raspberry Pi OS typically) — and then running IIAB's installer, to install everything (software AND content) on the external drive:

   ```
   curl iiab.io/install.txt | bash
   ```

Further / Advanced Details:

- #3106
- #3247

**Reply by derspion318** [reply]
I don't know why the thought of this didn't cross my mind. I blame it on lack of coffee. lol. I simply copied everything in /library to the root of the external then used /etc/fstab to mount the drive to /library. BOOM! Instant 4TB of space in the library. Thanks for jogging my memory!

**Comment by jvonau** [comment]
Not an answer but related to #3051

**Comment by holta** [comment]
Are there concrete suggestions to improve "external drive" guidance here?

https://wiki.iiab.io/go/FAQ#Can_I_permanently_attach_an_external_USB_drive%2C_to_add_more_content%3F

It's certainly a work-in-progress (hopefully getting better this year especially!) now that many more people are trying these important + ambitious new approaches :heavy_check_mark: 

ASIDE:

- #3351
